src/components/pane-layout/layout-manager.jsx

```diff
@@ -0,0 +1,185 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+import GoldenLayout from 'golden-layout'
+
+const config = {
+  settings: {
+    showPopoutIcon: false,
+    showCloseIcon: false,
+    showMaximiseIcon: false,
+  },
+  dimensions: {
+    dragProxyWidth: 300,
+    dragProxyHeight: 0,
+  },
+  content: [{
+    type: 'row',
+    content: [
+      {
+        type: 'column',
+        content: [
+          {
+            title: 'Editor',
+            type: 'react-component',
+            component: 'Positioner',
+            props: { positionerId: 'EditorPositioner' },
+            isClosable: false,
+          },
+          {
+            type: 'stack',
+            content: [
+              {
+                title: 'Console',
+                type: 'react-component',
+                component: 'Positioner',
+                props: { positionerId: 'ConsolePositioner' },
+                isClosable: false,
+              },
+              {
+                title: 'Workspace',
+                type: 'react-component',
+                component: 'Positioner',
+                props: { positionerId: 'WorkspacePositioner' },
+                isClosable: false,
+              },
+              {
+                title: 'App Info',
+                type: 'react-component',
+                component: 'Positioner',
+                props: { positionerId: 'AppInfoPositioner' },
+                isClosable: false,
+              },
+            ],
+          },
+        ],
+      },
+      {
+        title: 'Report',
```

Comment:
I think this should be called "Preview" and note "Report". It's a more accurate description of what it is, and what its used for - show a preview of the final document.

Reply:
sure sounds good